recipe/meta.yaml

```diff
@@ -11,22 +11,25 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  # kaleido-core=0.2.1 isn't available on s390x or ppc64le, see https://github.com/AnacondaRecipes/kaleido-core-feedstock/blob/d512c9bd10c86d252a6dfc3f0f9057e64ff92fba/recipe/meta.yaml#L26
+  skip: true  # [linux and (s390x or ppc64le)]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
+    - setuptools
```

Comment:
will need to address whether python build packages really do not need pinning, as we have opposing opinions on this.

Reply:
Here is recipe standards about python build tools and pinning https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/02_Recipes/Recipe_standards.md#the-requirements-section

Reply:
To me it's still unclear whether we are observing those recipe standards or if we ought to change them.